guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -201,6 +201,16 @@ endif::[]
 # journalctl -u leapp_resume -f
 ----
 
+ifdef::satellite[]
+. Complete the post-upgrade steps described in https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/upgrading_from_rhel_7_to_rhel_8/verifying-the-post-upgrade-state-of-the-rhel-8-system_upgrading-from-rhel-7-to-rhel-8[Verifying the post-upgrade state of the RHEL 8 system] in the _Upgrading from RHEL 7 to RHEL 8_ guide.
+. If you require SELinux to be in enforcing mode, run the following command before changing SELinux to enforcing mode:
++
+----
+# dnf reinstall foreman-selinux katello-selinux --disableplugin=foreman-protector
+----
```

Comment:
```suggestion
endif::[]
. If you require SELinux to be in enforcing mode, run the following command before changing SELinux to enforcing mode:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
ifdef::foreman-el[]
# dnf reinstall foreman-selinux --disableplugin=foreman-protector
endif::[]
ifndef::foreman-el[]
# dnf reinstall foreman-selinux katello-selinux --disableplugin=foreman-protector
endif::[]
----
ifdef::satellite[]
```

I think we can render this for Foreman and Katello too, without the links to the RH docs